subprojects/android-test/ui-testing-core/src/main/kotlin/com/avito/android/test/util/Instrumentations.kt

```diff
@@ -1,30 +1,14 @@
 package com.avito.android.test.util
 
 import android.app.Activity
-import android.os.Looper
-import androidx.test.platform.app.InstrumentationRegistry
-import androidx.test.runner.lifecycle.ActivityLifecycleMonitorRegistry
-import androidx.test.runner.lifecycle.Stage
+import com.avito.android.instrumentation.ActivityProviderFactory
+
+private val activityProvider = ActivityProviderFactory.create()
 
 fun getCurrentActivity(): Activity {
     return getCurrentActivityOrNull() ?: throw IllegalStateException("Resumed activity not found")
 }
 
 fun getCurrentActivityOrNull(): Activity? {
```

Comment:
Can we remove this method? 
It doesn't hide much. `ActivityProvider` is already hidden by an interface.

Reply:
removed